google-cloud-core/src/main/java/com/google/cloud/GcpLaunchStage.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * A qualifier indicating what level of access and support can be expected of a particualr Google
+ * Cloud Platform feature. The "GeneralAvailability" qualifier is not provided since the vast
+ * majority of features are in this category.
+ *
+ * <p>See more at the <a href="https://cloud.google.com/terms/launch-stages">Launch Stages Documentation</a>.
```

Comment:
Also add a general note that any particular feature might be in a later launch stage than annotated in the client library (i.e. the client library will lag a little)